docs/root/configuration/http/http_filters/composite_filter.rst

```diff
@@ -0,0 +1,24 @@
+.. _config_http_filters_composite:
```

Comment:
Can you cover this topic in arch overview also somewhere just so we have cross linking for the concepts?

Reply:
Added a section to http_filters.rst